tests/luatest.lua

```diff
@@ -221,6 +221,15 @@ local function testCanonicalData()
     checkReadBuffer(wireData)  
 end    
     
+local function testCreateEmptyString()
+    local b = flatbuffers.Builder(0)
+    local str = b:CreateString("")
+    monster.Start(b)
+    monster.AddName(b, str)
+    b:Finish(monster.End(b))
+    b:Output()
```

Comment:
Can you add an assertion that the data is read out correctly?

Reply:
Done

Reply:
Thanks!